app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for person in people:
+        Person(person["name"], person["age"])
+
+    for i in range(len(people)):
+        current_person = people[i]
+        wife_name = current_person.get("wife")
+        husband_name = current_person.get("husband")
+
+        target_person = Person.people[current_person.get("name")]
+
+        if wife_name is not None:
+            target_person.wife = Person.people[wife_name]
+        elif husband_name is not None:
+            target_person.husband = Person.people[husband_name]
```

Comment:
You could make your code shorter avoiding `is not None` comparison and making variable assignment during an if condition
```suggestion
    for person in people:
        target_person = Person.people[person.get("name")]

        if wife_name := person.get("wife"):
            target_person.wife = Person.people[wife_name]
        elif husband_name := person.get(""husband):
            target_person.husband = Person.people[husband_name]
```

Reply:
Thanks! that's very useful